frontend/src/index.js

```diff
@@ -1,49 +1,57 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import ReactDOM, { render } from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import axios from "axios";
 
-//default class component
-class MyClass extends React.Component {
-  render() {
-    return (
-      <h1>Hello World</h1>
-    );
+//Old controller prototype
+class ArticleController1 extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      feed: [],
+      trueFeed: [],
+      falseFeed: [],
+    }
   }
-}
 
-//Test function component
-function FunComponent(props) {
-  const [userInput, setUserInput] = useState('');
-  const [textArea, setTextArea] = useState('');
-  const [sub, setSub] = useState('');
-  const [myArray, setMyArray] = useState([]);
-
-  const handleSubmit = (e) => {
-    setSub(userInput);
-    e.preventDefault();
+  componentDidMount(){
+    this.initializeFeed();
   }
 
-  const handleUserInputChange = (e) => {
-    const input = e.target.value;
-    setUserInput(input);
+  initializeFeed = () => {
+    /*
+    axios.get("http://localhost:8000/")
+      .then((res) => this.setState({ feed: res.data.feed}))
+      .catch((err) => alert(err.message));
+      */
   }
 
-  const handletextAreaChange = (e) => {
-    const input = e.target.value;
-    setTextArea(input);
+  renderList = () => {
+    if(!this.state.feed){
+      //feed is being fetched and should show loading
+      return <Article title="Loading Title" prediction = "Loading Prediction" probability="Loading Probability" body="Loading Body"url="https://google.com" /> ;
```

Comment:
If this Article is pointing to google.com, I think it might be better to redirect it to our page

Reply:
Makes sense, I'll change it.